src/Compilers/Test/Core/CommonTestBase.cs

```diff
@@ -521,8 +521,9 @@ internal static void VerifyParentOperations(SemanticModel model)
         private static void CollectParentOperations(IOperation operation, Dictionary<IOperation, IOperation> map)
         {
             // walk down to collect all parent operation map for this tree
-            foreach (var child in operation.Children.WhereNotNull())
+            foreach (var child in operation.ChildOperations)
             {
+                if (child is null) continue;
```

Comment:
>child is null [](http://example.com/codeflow?start=20&length=13)

When is `child is null`?

Reply:
Theoretically never, this is likely some old code from the original IOperation implementation. Should be safe to remove.